### PR TITLE
Updated tdnf and yum package manager selection

### PR DESCRIPTION
**Pull request by dthaluru**
Changed package manager from dnf to tdnf

Added assumeyes as additional argument to yum/tdnf install command
to automatically answer interactive questions as 'yes'

### CODE REVIEW

**Comment by sofar** [comment]
Ah, so, I didn't realize that tdnf is actually not the same as dnf.

Because it isn't the same, can we not effectively remove dnf support, but instead add `tdnf` as an option?

As for yum, patching the command with `--assumeyes` is fine, of course.

**Comment by dthaluru** [comment]
We can add tdnf support in additional to dnf. I will send new pull request with tdnf and yum changes.

**Comment by dthaluru** [comment]
I submitted new pull requests for changes. Closing the current pull request.

File: src/ccmodules/package_upgrade.c [hunk 1]
```diff
@@ -62,7 +62,7 @@ void package_upgrade_handler(GNode *node) {
 #elif defined(PACKAGE_MANAGER_YUM)
 		exec_task("/usr/bin/yum update");
```
Comment (devimc) [inline src/ccmodules/package_upgrade.c]: maybe this line needs "--assumeyes"

File: configure.ac [hunk 2]
```diff
@@ -136,21 +136,21 @@ AC_ARG_WITH([datadir],[AC_HELP_STRING([--with-datadir=DIR],
...
 if test "x$PKGMGR" = "xswupd" ; then
 	AC_DEFINE([PACKAGE_MANAGER_SWUPD], [1], [Define if swupd package manager selected])
 else
 	if test "x$PKGMGR" = "xyum" ; then
 		AC_DEFINE([PACKAGE_MANAGER_YUM], [1], [Define if yum package manager selected])
 	else
-		if test "x$PKGMGR" = "xdnf" ; then
-			AC_DEFINE([PACKAGE_MANAGER_DNF], [1], [Define if dnf package manager selected])
+		if test "x$PKGMGR" = "xtdnf" ; then
+			AC_DEFINE([PACKAGE_MANAGER_DNF], [1], [Define if tdnf package manager selected])
```
Comment (devimc) [inline configure.ac]: maybe PACKAGE_MANAGER_DNF needs to be renamed to PACKAGE_MANAGER_TDNF

File: src/ccmodules/package_upgrade.c [hunk 3]
```diff
@@ -62,7 +62,7 @@ void package_upgrade_handler(GNode *node) {
...
 #elif defined(PACKAGE_MANAGER_DNF)
-		exec_task("/usr/bin/dnf update --refresh");
+		exec_task("/usr/bin/tdnf update --refresh");
```
Comment (devimc) [inline src/ccmodules/package_upgrade.c]: but, I did not install tdnf in /usr/bin/   u_u
a solution for this could be to add something like this [configure.ac](https://github.com/clearlinux/clr-cloud-init/blob/master/configure.ac#L75)  ?

Comment (sofar) [inline src/ccmodules/package_upgrade.c]: what is the standard path of tdnf?

Comment (dthaluru) [inline src/ccmodules/package_upgrade.c]: /usr/bin/tdnf is standard path used in Photon OS, but user can easily configure the location.

File: configure.ac (hunk 2 above)
Comment (dthaluru) [inline configure.ac]: I have submitted another PR with right fix.
https://github.com/clearlinux/clr-cloud-init/pull/16
